MapboxVision/src/main/java/com/mapbox/vision/VisionManager.kt

```diff
@@ -201,6 +186,34 @@ object VisionManager : BaseVisionManager {
 
         this.videoSource = videoSource
         this.videoRecorder = videoRecorder
+
+        mapboxTelemetry = MapboxTelemetry(
+            application,
+            mapboxToken,
+            MAPBOX_TELEMETRY_USER_AGENT
+        )
+
+        if (!isTurnstileEventSent) {
+            mapboxTelemetry.push(
+                AppUserTurnstile(
+                    MAPBOX_VISION_IDENTIFIER,
+                    BuildConfig.VERSION_NAME
+                )
+            )
+            isTurnstileEventSent = true
+        }
+
+        nativeVisionManager.create(
+            telemetryEventManager = MapboxTelemetryEventManager(mapboxTelemetry),
+            telemetryImageSaver = telemetryImageSaver
+        )
+
+        sessionManager = SessionManager.Impl(
+            application, nativeVisionManager,
```

Comment:
```suggestion
            application, 
            nativeVisionManager,
```

`nativeVisionManager` to next line

Reply:
done